ansible_collections/arista/avd/examples/dual-dc-l3ls/README.md

```diff
@@ -0,0 +1,609 @@
+
+```yaml title="DC1.yml"
+    DC1_L3_LEAF2:
+      bgp_as: 65102
+      evpn_gateway:
+        evpn_l2:
+          enabled: true
+        evpn_l3:
+          enabled: true
+          inter_domain: true
+      nodes:
+        dc1-leaf2a:
+          id: 3
+          mgmt_ip: 172.16.1.103/24
+          uplink_switch_interfaces:
+            - Ethernet3
+            - Ethernet3
+          evpn_gateway:
+            remote_peers:
+              - hostname: dc2-leaf2a
+        dc1-leaf2b:
+          id: 4
+          mgmt_ip: 172.16.1.104/24
+          uplink_switch_interfaces:
+            - Ethernet4
+            - Ethernet4
+          evpn_gateway:
+            remote_peers:
+              - hostname: dc2-leaf2b
+```
+
+Finally, more of the same, but this time for the L2 leaf switches:
+
+```yaml title="DC2.yml"
+l2leaf:
+  defaults:
+    platform: vEOS-lab
+    uplink_interfaces: ['Ethernet1', 'Ethernet2']
+    spanning_tree_mode: mstp
+
+  node_groups:
+    DC1_L2_LEAF1:
+      uplink_switches: ['dc2-leaf1a', 'dc2-leaf1b']
+      nodes:
+        dc1-leaf1c:
+          id: 11
+          mgmt_ip: 172.16.1.251/24
+          uplink_switch_interfaces:
+            - Ethernet8
+            - Ethernet8
+
+    DC1_L2_LEAF2:
+      uplink_switches: ['dc2-leaf2a', 'dc2-leaf2b']
+      nodes:
+        dc1-leaf2c:
+          id: 12
+          mgmt_ip: 172.16.1.252/24
+          uplink_switch_interfaces:
+            - Ethernet8
+            - Ethernet8
+```
+
+As should be clear, an L2 leaf switch is much simpler than an L3 switch. Hence there are fewer settings to define.
+
+## Specifying network services (VRFs and VLANs) in the EVPN/VXLAN fabric
+
+This file is identic to the one provided in the previous example and VRFs and VLANs are configured on all devices, since no `tags` or `filter` are being used.
+
+It is important to consider to add the new leaves in the inventory as described in the [Inventory](#content-of-the-inventoryyml-file) section, so they will rely on this file to configure the network services.
+
+```yaml title="NETWORK_SERVICES.yml"
+--8<--
+examples/dual-dc-l3ls/group_vars/NETWORK_SERVICES.yml
+--8<--
+```
+
+## Specifying endpoint connectivity in the EVPN/VXLAN fabric
+
+After the previous section, all VRFs and VLANs across the fabric are now defined accross all devices. The `ansible-avd-examples/dual-dc-l3ls/group_vars/CONNECTED_ENDPOINTS.yml` file specifies the connectivity for all endpoints in the fabric (typically servers):
+
+```yaml title="CONNECTED_ENDPOINTS.yml"
+--8<--
+examples/dual-dc-l3ls/group_vars/CONNECTED_ENDPOINTS.yml
+--8<--
+```
+
+The configuration of the endpoints in DC2 is identic to the ones in DC1, so the only change resides in the name of the leaves used and their ports.
+
+## The playbook
+
+The playbook is also the same, as the actions to execute in the fabric are the same. It is important to validate the `hosts` variable in the playbook to include all ansible groups, otherwise errors can be thrown.
```

Comment:
```suggestion
The playbook is also the same, as the actions to execute in the fabric are the same. It is important to validate the `hosts` variable in the playbook to include all Ansible groups. Otherwise, errors may be seen.
```